examples/classification/readme.md

```diff
@@ -0,0 +1,73 @@
+# HierarQML
+
+This tutorial guides on using HierarQcal to create quantum circuit models for classification tasks. It uses the modern stack of Pytorch Lightning and PennyLane for classical and quantum machine learning.
+
+## Setup
+
+Create a python environment and install the required packages using
+
+```
+pip install -r requirements.text
+```
+
+Set the Python Path using `export PYTHONPATH=.` before executing any commands later on, or you will get errors like ModuleNotFoundError: No module named ''.
+
+## Project Structure
+
+To go through the main project structure, let's train a QCNN on GTZAN data.
+
+### Directory Structure
+
+```
+├── hierarqml
+│   ├── data
+│   │   ├── base_data_module.py
+│   │   ├── gtzan.py
+│   │   └── __init__.py
+│   ├── __init__.py
+│   ├── models
+│   │   ├── base.py
+│   │   ├── __init__.py
+│   │   └── qcnn.py
+│   └── utils.py
+├── notebooks
+│   └── temp.ipynb
+├── readme.md
+├── requirements.txt
+└── training
+    ├── ansatz.py
+    ├── __init__.py
+    ├── run_expt.py
+    └── util.py
+```
+
+The main code is is `hierarqml` and `training`. `training` is used to train and experiment with the models. Withing `hierarqml` we have `data`, and `models`. `notebooks` is used for exploration and demonstration purposes.
```

Comment:
Remove the double "is" and fix this: "Withing `hierarqml` we have `data`, and `models`. `notebooks`"